### PR TITLE
Clean up housing counselor page

### PR DESCRIPTION
**Pull request by anselmbradford**
IE11 complained about unexpected end tag.

## Changes

- Correctly indents template markup.
- Adds `%20` in spaces in email sharing link (see https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/legacy/templates/hud/housing_counselor.html#L23)
- Converts:
```
<link href='{% static 'hud/hud.css' %}' rel='stylesheet' ></link>
<link href='{% static 'hud/hud_print.css' %}' rel='stylesheet' ></link>
```
To:
```
<link href='{% static 'hud/hud.css' %}' rel='stylesheet'></link>
<link href='{% static 'hud/hud_print.css' %}' rel='stylesheet'></link>
```

## Testing

1. visit `/find-a-housing-counselor`

### CODE REVIEW

File: cfgov/legacy/templates/hud/housing_counselor.html [hunk 1]
```diff
@@ -13,198 +13,255 @@
...
-  <div class="share mini">
-    <a href="http://www.facebook.com/sharer.php?u=http://www.consumerfinance.gov/find-a-housing-counselor/&amp;t=Find%20a%20housing%20counselor" class="share-facebook noStyles"><img src="{% static 'nemo/_/img/icon_small_facebook.png' %}" alt="Share on Facebook"></a>
-    <a href="http://twitter.com/intent/tweet/?url=http://www.consumerfinance.gov/find-a-housing-counselor/&amp;via=CFPB" class="share-twitter noStyles"><img src="{% static 'nemo/_/img/icon_small_twitter.png' %}" alt="Share on Twitter"></a>
-    <a href="http://api.addthis.com/oexchange/0.8/forward/email/offer?url=http://www.consumerfinance.gov/find-a-housing-counselor/&amp;title=Find a housing counselor&amp;pubid=ra-4da354ee346886d2" class="share-email noStyles"><img src="{% static 'nemo/_/img/icon_small_email.png' %}" alt="Share via email"></a>
-  </div>
-  <article class="page">
-
+    </ul>
+    <div class="share mini">
+        <a href="http://www.facebook.com/sharer.php?u=http://www.consumerfinance.gov/find-a-housing-counselor/&amp;t=Find%20a%20housing%20counselor" class="share-facebook noStyles">
```
Comment (chosak) [inline cfgov/legacy/templates/hud/housing_counselor.html]: Since you're touching these templates, is it worth converting these links to https?

Comment (anselmbradford) [reply cfgov/legacy/templates/hud/housing_counselor.html]: Done! I converted them all on this page to https

**Comment by anselmbradford** [comment]
@chosak Any idea what the new URL for portal.hud.gov/hudportal/HUD?src=/ohc_nint should be?

**Comment by chosak** [comment]
@anselmbradford I'm not sure. Based on the text around that link ("list of nationwide HUD-approved counseling intermediaries"), perhaps [this](https://apps.hud.gov/offices/hsg/sfh/hcc/hcs.cfm)?

**Comment by anselmbradford** [comment]
> I'm not sure. Based on the text around that link ("list of nationwide HUD-approved counseling intermediaries"), perhaps this?

Should I update it to that or do we need approval from someone for content changes on this page?

**Comment by chosak** [comment]
Actually, re-reading that ("intermediaries"), maybe it's [this](https://www.hudexchange.info/programs/housing-counseling/intermediaries-shfa/)? We should check with stakeholders before changing this link, though. I'm not sure who that would be.

**Comment by anselmbradford** [comment]
@sephcoster any luck on finding who could approve a URL change in here?

**Comment by anselmbradford** [comment]
@chosak I don't think this PR needs to wait on the updated URL to be merged. It's already that way on prod. Does it otherwise look okay?

**Pull request merged**